developer-guide/index.md

```diff
@@ -120,9 +128,23 @@ In order for the **first distribution channel** to be created, you’ll be promp
 
 We’ll be gathering feedback from our custom-built _Pepperoni-app_, hence we decide to name this channel the _Pepperoni channel_.
 
-### Copying the JSON configuration from the form editor
+### Retrieving the JSON configuration
+
+We can now retrieve the JSON configuration of the form and channel we just created. In order to retrieve the configuration, we head to the *Channel setup* page where we have two options:
+
+1. we copy the JSON directly from this page. This way suites best scenarios where internet connection is limited
```

Comment:
- Capitalize start of sentence.
- Add `.` to end of line.
- Change `way` to `method`

Reply:
done